content/applications/productivity/discuss/chatter.rst

```diff
@@ -0,0 +1,407 @@
+
+Send messages
+=============
+
+Chatter *composers* can send messages to outside contacts, without having to leave the database, or
+open a different application. This makes it easy to communicate with potential customers in the
+*Sales* and *CRM* applications, or vendors in the *Purchase* app.
+
+To send a message, first navigate to a record. For example, to send a message from a *CRM*
+opportunity, navigate to :menuselection:`CRM app --> Sales --> My Pipeline` and click on the Kanban
+card of an opportunity to open it. Then, at the top-right above the chatter composer, click
+:guilabel:`Send message`.
+
+.. tip::
+   Press :command:`Ctrl + Enter` to send a message, instead of using the :guilabel:`Send` button.
+
+If any :ref:`followers <discuss/add-followers>` have been added to the record, they are added as
+recipients of the message.
+
+.. warning::
+   :ref:`Followers <discuss/add-followers>` of a record are added as recipients of a message
+   automatically. If a follower should **not** receive a message, they must be removed as a follower
+   before the message is sent.
+
+.. image:: chatter/send-message-followers.png
+   :align: center
+   :alt: A chatter composer preparing to send a message to the followers of a CRM opportunity and
+         the customer listed on the opportunity record.
+
+Expand full composer
+--------------------
+
+The chatter composer can be expanded to a larger pop-up window, allowing for additional
+customizations.
+
+To open the full composer, click the :guilabel:`expand` icon in the bottom-right of the composer
+window.
+
+.. figure:: chatter/chatter-expand-icon.png
+   :align: center
+   :alt: A chatter composer with emphasis on the expand icon.
+
+   The expand icon in a chatter composer.
+
+Doing this opens a :guilabel:`Compose Email` pop-up window. Confirm or edit the intended
+:guilabel:`Recipients` of the message. The :guilabel:`Subject` field auto-populates based on the
+title of the record, though it can be edited if desired.
+
+To use a template for the message, select it from the :guilabel:`Load template` drop-down.
+
+.. note::
+   The number and type of templates available vary based on the record the message is created from.
+
+Click :guilabel:`📎 Attachments` to add any files to the message, then click :guilabel:`Send`.
+
+Edit sent messages
+------------------
+
+Messages can be edited after they are sent, to fix typos, correct mistakes, or add missing
+information.
+
+To edit a sent message, click the :guilabel:`... (three dot)` icon menu to the right of the message.
+Then, select :guilabel:`Edit`. Make any necessary adjustments to the message.
+
+.. image:: chatter/chatter-edit.png
+   :align: center
+   :alt: The edit message option in a chatter thread.
+
+To save the changes, press :command:`Ctrl + Enter`. To discard the changes, press :command:`Escape`.
+
+.. note::
+   When messages are edited after they have been sent, an updated message is **not** sent to the
+   recipient.
+
```

Comment:
also admins can edit anybody's message. this section refers to user editing own user messages/*log notes*